php/commands/core.php

```diff
@@ -143,6 +151,10 @@ public function download( $args, $assoc_args ) {
 			unlink($temp);
 		}
 
+		if ( version_compare( $from_version, '4.4', '>=' ) && version_compare( $assoc_args['version'], '4.4', '<' ) ) {
```

Comment:
Although, we need to ensure we safely handle the cases where WordPress.org doesn't return checksums for a version (for whatever reason).